### ISSUE TITLE
508 - Label should be more descriptive on View Opportunity page for the "delete" option

### ISSUE THREAD

**Issue by margaretdumas** [openopps/openopps-platform#2289]
Jaws reads the "x" for deleting a discussion item as "number link".  This should be more descriptive of what it does.  

Also when using the tab key, the "x" is reached before the link to the left of it.  Should read from left to right.

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing.

**Comment by margaretdumas** [comment]
06/7/2018 - Mdumas - In testing....

**Comment by margaretdumas** [comment]
Mdumas - Moving to review.  
Review notes: 
Without the screen reader software you can only review the tab order was updated on the view opportunity page where there are discussion items.  The tab order was getting to the "x" for deleting the comment before getting to the comment.  That has been correct so that the reader reads the content first then the delete comment option.  

The "x" was initially being read by Jaws as "number link" now it reads as "delete comment".

**Comment by kbwagner** [comment]
KWagner tested 6/7/18 in Chrome.  
@LisaLNelson ready for final review.

**Comment by LisaLNelson** [comment]
Approved.
